lib/src/_shared/PickerModalDialog.tsx

```diff
@@ -7,21 +7,48 @@ import Dialog, { DialogProps } from '@material-ui/core/Dialog';
 import { makeStyles } from '@material-ui/core/styles';
 import { DIALOG_WIDTH, DIALOG_WIDTH_WIDER } from '../constants/dimensions';
 
-export interface ModalDialogProps extends DialogProps {
-  onAccept: () => void;
-  onDismiss: () => void;
-  onClear: () => void;
-  onSetToday: () => void;
-  okLabel?: React.ReactNode;
-  cancelLabel?: React.ReactNode;
-  clearLabel?: React.ReactNode;
-  todayLabel?: React.ReactNode;
+export interface ExportedPickerModalProps {
+  /**
+   * "OK" button text
```

Comment:
Maybe should make a sentence.

```suggestion
   * "OK" button text.
```